python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -113,8 +142,9 @@ def beginJob(self):
             jesUncertainty_label = jesUncertainty
             if jesUncertainty == 'Total' and len(self.jesUncertainties) == 1:
```

Comment:
Does this line still work with the changes in this PR? If I'm running with hemfix enabled and want total JES unc, the JetCorrectorParameters constructor a few lines below segfaults. I believe this is because in this case `len(self.jesUncertainties)` is `2`, which means that `jesUncertainty_label` will not be modified here. If `jesUncertainty` is `'Total'`, then `jesUncertainty_label` will also be `'Total'`, while it should be `''`. 

Long story short, it seems to me the that the line should be:

```python
if jesUncertainty == "Total" and (len(self.jesUncertainties)==1 or (len(self.jesUncertainties)==2 and "HEMIssue" in self.jesUncertainties)):
                jesUncertainty_label = ""
```

Reply:
Indeed, I hadn't checked that particular combination... should be OK now.